### ISSUE TITLE
the time of training

### ISSUE THREAD

**Issue by Taocunguo** [yinguanchun/SKCDF#1]
May I ask how long the normal training time is？

**Comment by yinguanchun** [comment]
> May I ask how long the normal training time is？

Thank you for your attention to our project. In my memory, it took about 3 hours on the Synapse dataset. Training 1500 epochs on the AMOS dataset takes more than 12 hours, but in fact, the best results may have been achieved on the validation set very early.

**Comment by wangran20021226-crypto** [comment]
Hello, may I ask why the Synapse dataset you processed is only 468MB, which is much smaller than the original dataset?
